src/vision/VisionProcessor.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef VISION_PROCESSOR_H
+#define VISION_PROCESSOR_H
+#include <ros/ros.h>
```

Comment:
I already knew what the difference was it was just a question of whether or not it was in the library search path for the cmake build.